wikimedia/wikipedia-ios
# Add accessibility labels for notifications icon and filters

**Phabricator:** Partially addresses https://phabricator.wikimedia.org/T288776

### Notes
* This PR adds some accessibility labels to the notifications icon and the filters on the Notifications Center, to provide better context to VoiceOver users.
* The strings used here are also open to discussion, suggestions to improve the wording and context are appreciated

### Test Steps
1. Enable VoiceOver on the device - you can also use the accessibility inspector on Xcode
2. Tap the notifications icon on the Explore feed, both with and without the badge. Make sure that it correctly informs the user when the icon has the badge
3. On the Notifications Central, check if the user is correctly informed about the filters' states correctly, with and without filters engaged.
